RTN-008.tex

```diff
@@ -62,114 +63,113 @@
-
-Given the potentially high impact of the science, its visibility in the community, and importance to the NSF, we suggest that the operations project consider special handling of these TOO observations in the early operations era.  
-(Similar considerations apply in commissioning.) 
+Where LSST templates exist in the area to be covered, standard LSST processing can proceed as normal (subject to the caveats described in \citeds{DMTN-065}), and community alert brokers will provide community access to 60-second latency alerts from the TOO observations.
 
-The following analysis assumes that LSST TOO observations are relatively uncommon in this period (restricted to no more than 1 trigger/month, on average, perhaps) such that the staff impacts are tolerable.  
-We assume that any triggers would be on relatively well-localized GW sources (a few LSST pointings), and so any additional data processing is well within operational compute margins.
-These assumptions are in broad agreement with the followup proposal submitted by \citet{2018arXiv181204051M} in their response to the LSST call for cadence  white papers.
+Given the potentially high impact of the science, its visibility in the community, and importance to the NSF, we suggest that the project plan for special handling of GW TOO observations in the commissioning and early operations era.  
 
-\section{Recommendations}
+\section{Constraints} \label{sec:constraints}
 
-\subsection{Expedite data availability} \label{sec:data}
+Rubin's automated Alert Production relies on coadded templates generated from prior LSST images.  
+Given the early stage of Rubin commissioning during O4, limited on-sky time means it is probable that many LVK triggers will occur in sky areas Rubin has never imaged before.
+Without templates, then, fully automated, survey-grade Alert Production will not be possible. 
+Accordingly, bespoke custom processing will be required in order to generate timely and useful data products.
+Followup observations in the first 24 hours are among the most critical for identifying EM counterparts and constraining progenitor scenarios. 
 
-User access to raw and processed images is not required prior to 24 hours after the data are taken (\texttt{L1PublicT}; \texttt{LSR-REQ-0104} \citedsp{LSE-29}).  
-Moreover, \texttt{LSR-REQ-0126} specifies that images and catalog products except for alerts are embargoed for a \textit{minimum} of \texttt{L1PublicTmin} = 6 hours before release to anyone \citedsp{LSE-29}.
-Followup observations in the first 24 hours are among the most critical for constraining progenitor scenarios, so delays of this duration are unlikely to be viewed as acceptable by the community, particularly if they are not driven by technical limitations.
+Raw and processed LSST images are initially embargoed in both commissioning and operations.
```

Comment:
Do you plan to get into 30 days and 80 hrs at the PCW? We should discuss with Zeljko. I am not sure we want it to come up first in your session. But if you feel it will need to come up then Zeljko and I should bring it up generally first.